src/test/scala/chiselTests/VerificationSpec.scala

```diff
@@ -76,81 +57,19 @@ class VerificationSpec extends ChiselPropSpec with Matchers {
-      VerifAnnotation.annotate(cov)
-      VerifAnnotation.annotate(assm)
-      VerifAnnotation.annotate(asst)
     }
 
     // compile circuit
     val testDir = new File("test_run_dir", "VerificationAnnotationTests")
-    (new ChiselStage).emitSystemVerilog(
-      gen = new AnnotationTest,
-      args = Array("-td", testDir.getPath)
-    )
-
-    // read in annotation file
-    val annoFile = new File(testDir, "AnnotationTest.anno.json")
-    annoFile should exist
-    val annoLines = scala.io.Source.fromFile(annoFile).getLines.toList
-
-    // check for expected verification annotations
-    exactly(3, annoLines) should include("chiselTests.VerifAnnotation")
-    exactly(1, annoLines) should include("~AnnotationTest|AnnotationTest>asst")
-    exactly(1, annoLines) should include("~AnnotationTest|AnnotationTest>assm")
-    exactly(1, annoLines) should include("~AnnotationTest|AnnotationTest>cov")
-
-    // read in FIRRTL file
-    val firFile = new File(testDir, "AnnotationTest.fir")
-    firFile should exist
-    val firLines = scala.io.Source.fromFile(firFile).getLines.toList
-
-    // check that verification components have expected names
-    (exactly(1, firLines) should include).regex("^\\s*cover\\(.*\\) : cov")
-    (exactly(1, firLines) should include).regex("^\\s*assume\\(.*\\) : assm")
-    (exactly(1, firLines) should include).regex("^\\s*assert\\(.*\\) : asst")
-  }
-
-  property("annotation of verification constructs with suggested name should work") {
-
-    /** Circuit that annotates a renamed verification nodes. */
-    class AnnotationRenameTest extends Module {
-      val io = IO(new Bundle {
-        val in = Input(UInt(8.W))
-        val out = Output(UInt(8.W))
-      })
-      io.out := io.in
-
-      val goodbye = chisel3.assert(io.in === 1.U)
-      goodbye.suggestName("hello")
-      VerifAnnotation.annotate(goodbye)
-
-      VerifAnnotation.annotate(chisel3.assume(io.in =/= 2.U).suggestName("howdy"))
-    }
-
-    // compile circuit
-    val testDir = new File("test_run_dir", "VerificationAnnotationRenameTests")
-    (new ChiselStage).emitSystemVerilog(
-      gen = new AnnotationRenameTest,
-      args = Array("-td", testDir.getPath)
-    )
-
-    // read in annotation file
-    val annoFile = new File(testDir, "AnnotationRenameTest.anno.json")
-    annoFile should exist
-    val annoLines = scala.io.Source.fromFile(annoFile).getLines.toList
-
-    // check for expected verification annotations
-    exactly(2, annoLines) should include("chiselTests.VerifAnnotation")
-    exactly(1, annoLines) should include("~AnnotationRenameTest|AnnotationRenameTest>hello")
-    exactly(1, annoLines) should include("~AnnotationRenameTest|AnnotationRenameTest>howdy")
+    ChiselStage.emitSystemVerilogFile(gen = new AnnotationTest, args = Array("-td", testDir.getPath))
 
     // read in FIRRTL file
-    val firFile = new File(testDir, "AnnotationRenameTest.fir")
+    val firFile = new File(testDir, "AnnotationTest.sv")
```

Comment:
this isn't a .fir file, so should we rename this variable?

Reply:
Will do